xml/SLES-JeOS-12SP4-Quickstart.xml

```diff
@@ -323,58 +309,49 @@
 
   <itemizedlist>
    <listitem>
-    <para>JeOS for KVM and Xen</para>
+    <para>JeOS for KVM and Xen (Full Virtualized)</para>
```

Comment:
`Fully Virtualized`, maybe write `HVM`/`PVHVM` here (or PVH, I don't remember what the right one of the dozens of Xen modes is called)?